app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs

```diff
@@ -0,0 +1,15 @@
+{{#unless this.hidden}}
+  <div class="row bootstrap-mode-notice">
+    <div class="alert alert-info alert-bootstrap-mode">
+      <div class="col-text">
+        {{this.message}}
+      </div>
+      <div class="col-button">
+        <div class="button"><DButton @icon="user-plus" @href="/my/invited" @class="btn-primary bootstrap-invite-button" @label="bootstrap_mode_banner.invite_button_title" /></div>
```

Comment:
The `.button` and `.link` class names are a little too generic, can we use something more specific just to avoid any potential clashes with custom themes/plugins? Maybe `.alert--button` and `.alert--link`?

Alternatively, we can remove the div wrappers, and just style the button and the link directly, they have specific classes in `bootstrap-invite-button` and `bootstrap-wizard-link`.

Reply:
Okay, I've updated the class names to `.alert--button` and `.alert--link`. I need the div wrapper to center align the button and link.